lib/miq_automation_engine/models/miq_ae_yaml_import_gitfs.rb

```diff
@@ -0,0 +1,86 @@
+class MiqAeYamlImportGitfs < MiqAeYamlImport
+  def initialize(domain, options)
+    super
+    @fn_flags = File::FNM_CASEFOLD | File::FNM_PATHNAME
+    load_repo
+  end
+
+  def load_repo
+    raise MiqAeException::FileNotFound, "GIT repo dir: #{@options['git_dir']} not found" \
```

Comment:
If you have to go multiline anyway, then don't use an inline `unless`...instead:

```ruby
unless Dir.exist?(@options['git_dir'])
  raise MiqAeException::FileNotFound, "GIT repo dir: #{@options['git_dir']} not found"
end
```